package.json

```diff
@@ -68,7 +69,7 @@
     "ts-jest": "^29.1.5",
     "typedoc": "0.26.2",
     "typescript": "5.5.2",
-    "typescript-eslint": "^8.0.0-alpha.33",
+    "typescript-eslint": "^8.0.0-alpha.39",
```

Comment:
could we use the latest alpha version .40 here too?

Reply:
@Thilo-Acrolinx They only have 39 as of now
https://www.npmjs.com/package/typescript-eslint?activeTab=versions

![Screenshot 2024-07-03 at 17 19 32](https://github.com/acrolinx/sdk-js/assets/17494707/de054e6c-da29-43b4-b056-83918bb58e85)